platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -406,7 +406,7 @@ public void updateForLocation(
       for (Map.Entry<String, Object> configEntry : configEntries) {
         String configEntryKey = configEntry.getKey();
         Object configEntryValue =
-            sanitizeUIConfiguration(pid, configEntryKey.toLowerCase(), configEntry.getValue());
+            sanitizeUIConfiguration(pid, configEntryKey, configEntry.getValue());
```

Comment:
❓ Should we keep the `toLowerCase()` here and add `Locale.getDefault()` to it cause it's used beyond just the comparison? I'm not sure what the answer is just a question.

Reply:
I'm going back and forth on it. Wonder what @Lambeaux or @coyotesqrl thinks

Reply:
I think it's correct they way you have it now.

Reply:
Agreed. The way you have it now should work, as far as I can tell.